docs/image-config/README.md

```diff
@@ -0,0 +1,142 @@
+# Configuring the K8ssandra images
+
+Often users find it neccesary to configure various properties of the images they are using, this may be to:
+
+* Use an internal image registry to avoid dependancy on Docker Hub.
+* Use a custom image (for example to customise aspects of the classpath or run customised versions of Cassandra).
+
+## cass-operator images
+
+The cassandra pods have containers for Cassandra (including the management API), Medusa, config-builder and system-server-logger. These pods are under the management of cass-operator.
+
+cass-operator allows the user to provide a [`ImageConfig`](https://github.com/k8ssandra/cass-operator/blob/master/config/manager/image_config.yaml) to the operator via a ConfigMap, which contains a yaml representation of an `ImageConfig` specifying image sources. 
+
+The location of the mounted ConfigMap is then referenced in the [`OperatorConfig`](https://github.com/k8ssandra/cass-operator/blob/master/config/manager/controller_manager_config.yaml#L1).
+
+It is important to note that all image configurations occur in the ConfigMap mounted into the cass-operator deployment, not via the K8ssandraCluster or CassandraDatacenter CRs. Even though the `ImageConfig` looks like a Kubernetes API extension (and even has a `Kind`, and `ApiVersion`), **it is not**.
```

Comment:
CassandraDatacenter allows to specify the the image configurations, so why say this here?

Reply:
Talking about CRDs as "API extension" feels odd. Why not simply say CRD?

Reply:
Not to mention, what value does that clause bring?

Reply:
> Not to mention, what value does that clause bring?

Avoiding the immense confusion that I experienced when I saw something that looked like a CR when it isn't a CR.

> CassandraDatacenter allows to specify the the image configurations, so why say this here?

You're right that it needs to be re-worded. I'll look at making it clearer.